iceoryx_posh/include/iceoryx_posh/roudi/introspection_types.hpp

```diff
@@ -61,11 +61,11 @@ const capro::ServiceDescription IntrospectionPortService(INTROSPECTION_SERVICE_I
 /// @todo if future fixed string is aligned to 8 byte, the alignment here can be removed
 struct PortData
 {
-    alignas(8) cxx::CString100 m_name;
-    alignas(8) cxx::CString100 m_caproInstanceID;
-    alignas(8) cxx::CString100 m_caproServiceID;
-    alignas(8) cxx::CString100 m_caproEventMethodID;
-    alignas(8) cxx::CString100 m_runnable;
+    alignas(8) ProcessName_t m_name;
```

Comment:
Should we have a convention for the string types ?
Here we have `IdString` and `RunnableName_t` which are both string aliases.

Reply:
@ithier we should probably have that for all aliases. From my point of view, it's out of scope for this PR though.

Reply:
@elBoberido look at the comment in line 61. Could we remove the `alignas(8)` therefore or do I miss something?!

Reply:
you are right